### PR TITLE
SC-499 SC-500: Update docs for My First Realm and My First Application

### PR DESCRIPTION
**Pull request by harrycguo**
### Your checklist for this pull request

Thank you for your contribution to the `next-dev-docs` repo. 

🚨Please review the [Pull request guidelines](../contributor-guide/contributor-guide.md#pull-request-guidelines) to this repository before submitting this PR.

- [ ] Your code builds clean without any errors or warnings
  
  ```bash
  yarn build
  ```

- [ ] You are using approved terminology

  Refer to the [style guide](../contributor-guide/style-guide.md) for help.

### Description 

Closes: https://beyondidentity.atlassian.net/browse/SC-499 https://beyondidentity.atlassian.net/browse/SC-500

Updates the getting started guides since we create My First Realm and My First Application now. The following changes were made:

Getting Started React:
-Navigation to Beyond Identity Admin Realm to get Management API Token
-Added tip to skip realm creation since we've already created My First Realm

Getting Started Angular:
-Navigation to Beyond Identity Admin Realm to get Management API Token
-Added tip to skip realm creation since we've already created My First Realm

Getting Started NextJS:
-Added tips and copys to skip realm creation and application creation since we've already created My First Realm and My First Application

### CODE REVIEW

File: docs/get-started/getting-started-nextjs.mdx [hunk 1]
```diff
@@ -71,69 +71,78 @@ git --version
...
-  ```bash
-  git clone https://github.com/nextauthjs/next-auth-example.git
-  cd next-auth-example
-  npm install
-  ```
+```bash
+git clone https://github.com/nextauthjs/next-auth-example.git
+cd next-auth-example
+npm install
+```
```
Comment (Will5) [inline docs/get-started/getting-started-nextjs.mdx:78]: nit: Indents after lists are usually intentional, as they affect how content is displayed. I presume a prettifier did this automatically, but I typically undo these changes before committing and pushing.

Example:
| [https://developer.beyondidentity.com](https://developer.beyondidentity.com/docs/next/get-started-nextjs) | [https://developer-docs-git-harry-sc-499-500-beyondidentity.vercel.app](https://developer-docs-git-harry-sc-499-500-beyondidentity.vercel.app/docs/next/get-started-nextjs) |
|--------|--------|
| Screenshot 2023-08-28 at 5 19 09 PM | Screenshot 2023-08-28 at 5 19 04 PM |

File: docs/get-started/getting-started-angular.mdx [hunk 2]
```diff
@@ -119,21 +119,25 @@ APP_CLIENT_SECRET=<app-client-secret>
...
 
   ![Admin Console Beyond Identity Admin realm](../images/apps-beyond-identity-management-api.png)
 
-2. Select the **API Tokens** tab and click **Create a token**. 
+2. Select the **API Tokens** tab and click **Create a token**.
 
 3. Enter a name for the token and click **Create token**.
 
-4. In the confirmation message, copy the API token and click **Close this message**. 
+4. In the confirmation message, copy the API token and click **Close this message**.
```
Comment (Will5) [inline docs/get-started/getting-started-angular.mdx:140]: nit: We might as well convert these to `1`'s, like you did in `getting-started-react`, if only for consistency. The only time this is not useful is if an include also has a numbered list, which can throw off docusaurus's count.

File: docs/get-started/getting-started-nextjs.mdx (hunk 1 above)
Comment (harrycguo) [reply docs/get-started/getting-started-nextjs.mdx]: Okay i reverted, but I was wondering - does it look better when it is flush on the left? Pretty much every other instance of it has it left aligned.

Screenshot 2023-08-29 at 1 36 03 PM

Comment (Will5) [reply docs/get-started/getting-started-nextjs.mdx]: Sorry, I didn't mean to suggest that was the only instance. Patricia was strongly for the indents, aligning everything to the bullets/numbers. I can go either way, but I do think we should be consistent across the docs site, so I've maintained the indents.

**Comment by ismailmustafa** [comment]
I'm wondering if we should be a bit more aggressive here and outright remove those two sections (create a realm, create an application). Alternatively, if we really want to keep them in, can we collapse them automatically and suggest they open it up only if they need to follow those directions? Right now, it's very easy for me to skip over the note.

**Comment by harrycguo** [comment]
@ismailmustafa @Will5 I've updated the Getting Started NextJS guide with collapsible sections for creating a new realm and new application. I left it be in the React + Angular JS sections since the steps were a bit weird to work around and didn't think it was as big of a deal there.

File: docs/includes/_developer-sign-up-process.mdx [hunk 3]
```diff
@@ -10,12 +10,14 @@ import DevSignUpFlow from '../images/sign-up-flow.png';
...
\ No newline at end of file
+* A <a href="#" data-tooltip="A public and private key pair. The private key is generated and stored and never leaves the user’s devices’ hardware root of trust. The public key, on the other hand, is sent to the Beyond Identity cloud. All keys are cryptographically linked to the user and can be centrally managed using our APIs."><b>Universal Passkey</b></a> for accessing the Admin Console, bound to your identity and stored on your device or browser. You'll be logged into the Admin Console automatically during this process step. You'll log in without a password when logging into the console from the same device or browser.
+
+* A bootstrap Realm and Application called **My First Realm** and **My First Application**, to be used with our [Getting Started with NextJS](/docs/next/get-started-nextjs) guide.
```
Comment (Will5) [inline docs/includes/_developer-sign-up-process.mdx:23]: Same question.

File: docs/get-started/getting-started-angular.mdx [hunk 4]
```diff
@@ -218,31 +228,35 @@ import CreateRealmConsole from '../includes/_create-realm-console.mdx';
 
 ### Create an application
 
-In this step, you'll create a new Application that will contain the configuration for your users.
+In this step, you'll create a new Application that will contain the configuration for your authentication.
+
+:::caution note:
+You will need to create a new Application. **My First Application** is to be used only with our [**Getting Started with NextJS**](/docs/next/get-started-nextjs) guide.
+:::
```
[the remark this answers is not in the document]
Comment (harrycguo) [reply docs/get-started/getting-started-angular.mdx:235]: My First Application uses Hosted Web as type of authenticator config, these guides use the embedded SDK.

File: docs/includes/_developer-sign-up-process.mdx (hunk 3 above)
Comment (harrycguo) [reply docs/includes/_developer-sign-up-process.mdx:23]: Ditto

File: docs/get-started/getting-started-angular.mdx [hunk 5]
```diff
@@ -190,14 +194,20 @@ In this step, you'll create a set of Beyond Identity resources containing your e
 
 ### Create a new realm
 
-So far, you have the Admin Realm for your Beyond Identity tenant only. In this step, you'll create a new Realm to hold your users' identities.  
+So far, you have the Admin Realm for your Beyond Identity tenant only. In this step, you'll create a new Realm to hold your users' identities.
+
+:::note
+You can skip straight to **Step 4** below, as we've already created **My First Realm** for you. But if you want to go through the steps of creating a new realm yourself, follow **Steps 1 thru 3** below.
```
Comment (turnipdabeets) [inline docs/get-started/getting-started-angular.mdx:200]:
```suggestion
You can skip straight to **Step 4** below, as we've already created **My First Realm** for you. But if you want to go through the steps of creating a new realm yourself, follow **Steps 1 - 3** below.
```

File: docs/get-started/getting-started-angular.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (turnipdabeets) [reply docs/get-started/getting-started-angular.mdx:235]: maybe we can add a sentence here to clarify this, something like: "**My First Application** has been configured specifically for the NextJS getting started guide that uses our Hosted Web Authenticator"

[the remark this answers is not in the document]
Comment (harrycguo) [reply docs/get-started/getting-started-angular.mdx:235]: Added!

**Review by turnipdabeets — APPROVED** [review]

**Pull request merged**